packages/link/Link.js

```diff
@@ -3,32 +3,54 @@ import PropTypes from 'prop-types';
 import { isAbsoluteUrl } from '@availity/resolve-url';
 
 // if absolute or loadApp is disabled, return url. otherwise loadappify the url
-export const getUrl = (url = '', loadApp) => {
-  const absolute = isAbsoluteUrl(url);
+export const getUrl = (url = '', loadApp, absolute) => {
   if (absolute || !loadApp) return url;
 
   return `/public/apps/home/#!/loadApp?appUrl=${encodeURIComponent(url)}`;
 };
 
+// takes href and transforms it so that we can compare hostnames and other properties
+const getLocation = href => {
+  const location = document.createElement('a');
+  location.href = href;
+  return location;
+};
+
 const AvLink = ({
   tag: Tag,
-  href: url,
+  href,
   target,
   children,
   onClick,
   loadApp,
+  rel,
```

Comment:
rather than destructuring `rel` from props, leave it out, and if it's specified it will be spread onto the `Tag` and override whatever we set it to.